.gitignore

```diff
@@ -1,6 +1,9 @@
 # gitignore
 
 /node_modules
+/.idea
```

Comment:
please revert this file; these sorts of things should be in your personal global gitignore, not in every project you touch.

Reply:
fixed in latest commit.